roles/nova-common/tasks/ceph_integration.yml

```diff
@@ -0,0 +1,53 @@
+---
+- name: fetch rados.py
+  get_url: url={{ ceph.rados_url }}
+           dest="{{ openstack_source.virtualenv_base }}/nova/lib/python2.7/site-packages/rados.py"
+           owner=root
+           group=root
+           mode=0644
+  when: openstack_install_method == 'source'
+
+- name: fetch rbd.py
+  get_url: url={{ ceph.rbd_url }}
+           dest="{{ openstack_source.virtualenv_base }}/nova/lib/python2.7/site-packages/rbd.py"
+           owner=root
+           group=root
+           mode=0644
+  when: openstack_install_method == 'source'
+
+- name: fetch rados.py
+  get_url: url={{ ceph.rados_url }}
+           dest="{{ lookup('items', 'openstack_package.virtualenv_base') }}/nova/lib/python2.7/site-packages/rados.py"
+           owner=root
+           group=root
+           mode=0644
+  when: openstack_install_method == 'package'
+
+- name: fetch rbd.py
+  get_url: url={{ ceph.rbd_url }}
+           dest="{{ lookup('items', 'openstack_package.virtualenv_base') }}/nova/lib/python2.7/site-packages/rbd.py"
```

Comment:
Pretty sure you can set dest to `/opt/openstack/current/nova/lib/python2.7/site-packages/rbd.py` and not bother when a task for source and a task for package.